modules/imgproc/include/opencv2/imgproc.hpp

```diff
@@ -4730,6 +4730,20 @@ CV_EXPORTS_W Size getTextSize(const String& text, int fontFace,
                             double fontScale, int thickness,
                             CV_OUT int* baseLine);
 
+
+/** @brief Calculates the font-specific size to use to achieve a given height in pixels.
+
+@param fontFace Font to use, see cv::HersheyFonts.
+@param pixelHeight Pixel height to compute the fontScale for
+@param thickness Thickness of lines used to render the text.See putText for details.
+@return The fontSize to use for cv::putText
+
+@see cv::putText
+*/
+CV_EXPORTS_W double getFontScaleFromHeight(const int fontFace,
```

Comment:
no need to pass values by const

Reply:
Personally, I like to keep things const whenever possible. Can change it if you advise, though

Reply:
It is minor issue, but it leads to inconsistency between public functions, e.g. comparing to getTextSize, where only String parameter is passed by const ref.